### PR TITLE
Move proposals to a different page

### PR DESCRIPTION
**Pull request by vidya-ram**

### CODE REVIEW

File: funnel/templates/macros.html.jinja2 [hunk 1]
```diff
@@ -106,10 +106,79 @@
...
     {%- if project.hasjob_embed_url -%}<a class="sub-navbar__item mui--text-subhead {% if current_page == 'project' %}smooth-scroll{%- endif %}" href="{%- if current_page != 'project' -%}{{ project.url_for() }}{%- endif %}#related-jobs">Jobs</a>{%- endif %}
     {%- if project.primary_venue -%}<a class="sub-navbar__item mui--text-subhead {% if current_page == 'project' %}smooth-scroll{%- endif %}" href="{%- if current_page != 'project' -%}{{ project.url_for() }}{%- endif %}#venue">Venue</a>{%- endif %}
-    {%- if project.proposals -%}<a class="sub-navbar__item mui--text-subhead {% if current_page == 'project' %}smooth-scroll{%- endif %}" href="{%- if current_page != 'project' -%}{{ project.url_for() }}{%- endif %}#proposals">Proposals</a>{%- endif %}
+    {%- if project.proposals -%}<a class="sub-navbar__item mui--text-subhead {% if current_page == 'proposals' %}smooth-scroll{%- endif %}" href="{%- if current_page != 'proposals' -%}{{ project.url_for('view_proposals') }}{%- endif %}#proposals">Proposals</a>{%- endif %}
```
Comment (jace) [inline funnel/templates/macros.html.jinja2]: What is `current_page`? We can do this as `current_view.name` and make sure all the views have a `name` attribute.

File: funnel/templates/proposals.html.jinja2 [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    <div class="grid__col-xs-12 grid__col-md-5 grid__col-lg-4">
+      {%- if project.current_roles.admin or 'checkin-event' in g.permissions %}
+        {{ admin_panel(project) }}
+      {% endif %}
+    </div>
+
+    <div class="grid__col-xs-12 grid__col-md-7 grid__col-lg-8" id="proposals">
+      <div class="grid__col-xs-12">
+        <h2 class="mui--text-display1 mui--text-left project-section__headline">All proposals</h2>
+        {%- if 'view-status' in current_auth.permissions %}
```
Comment (jace) [inline funnel/templates/proposals.html.jinja2]: This should be `current_auth.permissions.view_status`. Backend code will need to rename `view-status` to `view_status`. We should also find a better name for it. Maybe `view_review_state`

File: funnel/templates/macros.html.jinja2 (hunk 1 above)
Comment (iambibhas) [reply funnel/templates/macros.html.jinja2]: This macro function renders header for 2 types of pages - project and schedule. `current_page` is passed to the function from templates. It's not exactly the name of the view. E.G. SessionView's `view` and ProjectScheduleView's `view` both use same `schedule.html.jinja2` template which pass `schedule` to this function. It's more like view type than view name.

File: funnel/templates/proposals.html.jinja2 [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+        <tr id="{{ proposal.id }}">
+          <td class='js-searchable'>
+            <h3 class="zero-bottom-margin"><a href="{{ proposal.url_for() }}" class="mui--text-title mui--text-bold nounderline">{{ proposal.title }}</a></h3>
+            <p>
+              {%- if proposal.speaker -%}
+                <span class="mui--text-subhead">{{ proposal.speaker.pickername }}</span>
+                {%- if proposal.user != proposal.speaker %}
+                  <small class="mui--text-caption mui--text-accent">via {{ proposal.user.pickername }}</small>
+                {%- endif %}
+                {%- if g.user and 'edit-project' in g.permissions %}
```
Comment (jace) [inline funnel/templates/proposals.html.jinja2]: `current_auth.permissions`

File: funnel/templates/proposals.html.jinja2 [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+            </p>
+            <ul class="mui-list--inline mui--text-body2 proposal_list--details list-item-rgborder">
+              {%- if proposal.project != project %}
+                <li>{{ proposal.project.short_title() }}</li>
+              {% elif proposal.section.title %}
+                <li>{{ proposal.section.title }}</li>
+              {% endif %}
+              <li>{{ proposal.technical_level }}</li>
+              <li>{{ proposal.voteset.count }} upvotes
+                {% if 'view-voteinfo' in g.permissions %}
```
Comment (jace) [inline funnel/templates/proposals.html.jinja2]: Here too. All these need to change.

File: funnel/templates/proposals.html.jinja2 [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+{%- endmacro %}
+
+{% block headline %}
+  {{ project_header(project, page_btns=page_btns(project), current_page='proposals') }}
+{% endblock %}
+
+{% block contentwrapper %}
+  <div class="grid details-section">
+    <div class="grid__col-xs-12 grid__col-md-5 grid__col-lg-4">
+      {%- if project.current_roles.admin or 'checkin-event' in g.permissions %}
```
Comment (jace) [inline funnel/templates/proposals.html.jinja2]: `current_auth`

**Pull request merged**